cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,46 @@
-describe('', () => {
+describe('CRUD article', () => {
+  let user;
+  let articleData;
   before(() => {
+    cy.visit(' ');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      articleData = generateArticle;
+    });
+  });
 
+  it('should create the article', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.visit('/editor');
+    cy.contains('.nav-link', 'New Article').click();
+    cy.get('[placeholder="Article Title"]')
+      .type(articleData.title);
+    cy.get('[placeholder="What\'s this article about?"]')
+      .type(articleData.description);
+    cy.get('[placeholder="Write your article (in markdown)"]')
+      .type(articleData.body);
+    cy.contains('.btn', 'Publish Article')
+      .click();
+    cy.url().should('contain', 'article/');
   });
+  it('should delete the article', () => {
+    cy.login(user.email, user.username, user.password);
```

Comment:
This login step seems to be repeated in multiple tests. Consider moving it to a `beforeEach` block to avoid repetition and make the tests more maintainable.